samples/HybridApp/HybridApp/WebUI/Pages/FetchData.razor

```diff
@@ -39,7 +39,9 @@ else
 
     protected override async Task OnInitializedAsync()
     {
-        if (Microsoft.MobileBlazorBindings.WebView.BlazorHybridHost.TryGetEmbeddedResourceFile("weather.json", out var stream))
+        // The static web asset is loaded from the wwwroot from _content/<AssemblyOrNugetName>/weather.json
+        // see https://docs.microsoft.com/en-us/aspnet/core/razor-pages/ui-class?view=aspnetcore-3.1&tabs=visual-studio#consume-content-from-a-referenced-rcl for more info
```

Comment:
Suggest removing the `en-us` from the URL so that it auto-picks the right locale for each user.

```suggestion
        // see https://docs.microsoft.com/aspnet/core/razor-pages/ui-class?view=aspnetcore-3.1&tabs=visual-studio#consume-content-from-a-referenced-rcl for more info
```